### PR TITLE
[#1583] fix: fix frontend build

### PR DESCRIPTION
**Pull request by MSzalowski**
## List of changes

- Fix building the frontend image.

## Checklist

- [related issue](https://github.com/IntersectMBO/govtool/issues/1583)
- [x] My changes generate no new warnings
- [x] My code follows the [style guidelines](https://github.com/IntersectMBO/govtool/tree/main/docs/style-guides) of this project
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the [changelog](https://github.com/IntersectMBO/govtool/blob/main/CHANGELOG.md)
- [ ] I have added tests that prove my fix is effective or that my feature works

### CODE REVIEW

File: govtool/frontend/package.json [hunk 1]
```diff
@@ -4,7 +4,7 @@
   "version": "1.0.8",
   "type": "module",
   "scripts": {
-    "build": "vite build",
+    "build": "NODE_OPTIONS=--max_old_space_size=8192 vite build",
```
Comment (MSzalowski) [inline govtool/frontend/package.json:7]: This might not be needed. Needs further testing.

File: govtool/frontend/vite.config.ts [hunk 2]
```diff
@@ -1,40 +1,25 @@
...
-          loader: "jsx",
-          jsx: "automatic",
-        });
-      },
-    },
     react(),
+    babel({
+      include: [
+        "node_modules/@intersect.mbo/**/*.js",
+        "node_modules/@intersect.mbo/**/*.jsx",
```
Comment (MSzalowski) [inline govtool/frontend/vite.config.ts:14]: This might not be needed. Needs further testing,

**Comment by MSzalowski** [comment]
That PR should be treated as a candidate for fix.

**Comment by MSzalowski** [comment]
TODO:
Verify dependent packages working such as:
usersnap and CSL

**Review by jdyczka — APPROVED** [review]

File: govtool/frontend/vite.config.ts (hunk 2 above)
Comment (MSzalowski) [reply govtool/frontend/vite.config.ts:15]: That is needed

File: govtool/frontend/package.json (hunk 1 above)
Comment (MSzalowski) [reply govtool/frontend/package.json]: Removed that.

**Pull request merged**